src/pages/TransactionPage/index.tsx

```diff
@@ -76,23 +89,35 @@ export default function TransactionPage() {
                 <TransactionStatus>{tx.status.__typename.replace('Status', '')}</TransactionStatus>
               </TransactionDataRow>
             ) : null}
-            <TransactionDataRow>
+            {/* <TransactionDataRow>
               <RowKeyColumn>Maturity:</RowKeyColumn>
               <RowValueColumn>{tx.maturity}</RowValueColumn>
-            </TransactionDataRow>
+            </TransactionDataRow> */}
             <TransactionDataRow>
               <RowKeyColumn>Gas Price:</RowKeyColumn>
               <RowValueColumn>
                 {parseToFormattedNumber(tx.gasPrice)} {BASE_COIN_NAME}
               </RowValueColumn>
             </TransactionDataRow>
+            <TransactionDataRow>
+              <RowKeyColumn>Byte Price:</RowKeyColumn>
+              <RowValueColumn>
+                {parseToFormattedNumber(tx.bytePrice)} {BASE_COIN_NAME}
+              </RowValueColumn>
+            </TransactionDataRow>
```

Comment:
Byte price is being removed in the next release, so don't even bother with it. It'll just confuse people. https://github.com/FuelLabs/fuel-specs/pull/333